docs/source/reference/config-examples.md

```diff
@@ -178,35 +181,84 @@ of the site as well as the applicable location call:
 
 ```bash
 server {
-	listen 80;
-	server_name NO_HUB.DOMAIN.TLD;
+    listen 80;
+    server_name NO_HUB.DOMAIN.TLD;
 
-	# Tell all requests to port 80 to be 302 redirected to HTTPS
-	return 302 https://$host$request_uri;
+    # Tell all requests to port 80 to be 302 redirected to HTTPS
+    return 302 https://$host$request_uri;
 }
 
 server {
-	listen 443;
-	ssl on;
+    listen 443;
+    ssl on;
 
-	# INSERT OTHER SSL PARAMETERS HERE AS ABOVE
+    # INSERT OTHER SSL PARAMETERS HERE AS ABOVE
```

Comment:
Let's clarify this a bit more. Perhaps: `NO_HUB.DOMAIN.TLD` is a different SSL cert than above, but configured in a similar fashion.